selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/CSharpFileEditingTest.java

```diff
@@ -56,69 +56,80 @@
   @Inject private SeleniumWebDriver seleniumWebDriver;
   @Inject private TestCommandServiceClient testCommandServiceClient;
   @Inject private CommandsPalette commandsPalette;
-
+  @Inject TestWorkspaceServiceClient testWorkspaceServiceClient;
   @Inject private Consoles consoles;
 
   @BeforeClass
   public void setUp() throws Exception {
     ide.open(workspace);
+    createDotNetAppFromWizard();
+    restoreLanguageServer();
+    projectExplorer.quickRevealToItemWithJavaScript(PROJECT_NAME + "/Program.cs");
+  }
+
+  @AfterMethod
+  public void restartWorkspace() throws Exception {
+    if (testWorkspaceServiceClient.getStatus(workspace.getId()).equals(WorkspaceStatus.RUNNING)) {
```

Comment:
What if workspace is not running at the moment? It doesn't mean that workspace is restarting at the moment, or language server will be restarted as well, do it?